src/resmom/mom_main.c

```diff
@@ -10039,7 +10045,8 @@ main(int argc, char *argv[])
 #ifdef PMIX
 	PMIx_server_finalize();
 #endif
-
+	destroy_tree(AVL_jobs);
+	unload_auths();
```

Comment:
It's not since one of our team is adding debug log messages in PBS, I thought this should be called before we close log. If you want I can revert it, but its harmless change.